src/Service/MailTemplateHelper.php

```diff
@@ -367,9 +368,27 @@ private function getMacroValues(MailTemplate $mailTemplate): array
     /**
      * @return array|ComplexMacro[]
      */
-    private function getComplexMacros(object $entity): array
+    private function getComplexMacros(object $entity, array $values): array
     {
-        return $this->macroHelper->buildMacros($entity);
+        $macros = [];
+
+        $linkKeys = ['board.email', 'board.url'];
+        foreach ($linkKeys as $key) {
+            $url = $values[$key] ?? null;
+            if (null !== $url) {
+                $text = $url;
+                if (filter_var($url, FILTER_VALIDATE_EMAIL)) {
+                    $url = 'mailto:'.$url;
+                }
+
+                $macros[$key.'.link'] = new ComplexMacro(
+                    $this->macroHelper->createLink($url, $text),
+                    $text
+                );
+            }
+        }
```

Comment:
All the occurrences of `url` confused me a little, but it looks like it should do the job!